notebooks/05_transform.ipynb

```diff
@@ -0,0 +1,3444 @@
+{
```

Comment:
<p><code>instead of sample throguh</code> &lt;- typo </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Thank you, I corrected</p>